content/strategy-goals/strategy/code-graph/code-insights/index.md

```diff
@@ -122,18 +123,19 @@ Users do not need to learn a new query language or format to use Code Insights.
 
 ## Where we are now
 
-Code Insights is a [Beta Feature](../../../../departments/product-engineering/product/process/beta_and_experimental_feature_labels.md). It entered Beta in August 2021, and will likely remain in Beta into Q4.
+Code Insights is a [Beta Feature](../../../../departments/product-engineering/product/process/beta_and_experimental_feature_labels.md). It entered Beta in August 2021, and will likely launch to General Availability in FY23Q1.
 
 This means that the Code Insights team is singularly focused on bringing Code Insights into general availability. To reach that goal, Code Insights will be building stability, features, and core product experiences based on the feedback we get from our beta participants.
 
-During the Beta, we are also evaluating pricing and packaging options. It is likely Code Insights will be a standalone feature with its own pricing.
+During the Beta, we are also evaluating pricing and packaging options. It is likely Code Insights will be a standalone feature with its own pricing. The current pricing and packaging proposal is [internally available](https://docs.google.com/document/d/11Y5ZDIT_nCwkobGzVgseM7vgmk5Hkt-4UZHvivHwN7A/edit#heading=h.tivmnwrtt3gg).
 
 For what we have learned so far and how we are operationalizing those learnings, see [Strategy](#strategy-and-plans).
 
 ### Top customer, support, sales and marketing issues
 
 - The biggest issues customers face, and that we're addressing, are the yet-to-be-built product gaps. See the docs page for [Current Limitations of Code Insights](https://docs.sourcegraph.com/code_insights/explanations/current_limitations_of_code_insights) for an up-to-date list.
-- Internally, the main task for Code Insights is determining the pricing and packaging
+- The main sales task for Code Insights is operationalizing the pricing and packaging
```

Comment:
I'm not sure about the value of adding tasks that other teams are doing here, I think it's harmless to include (so if it's valuable to you go for it) but this section is more intended as "what are support, sales, and marketing teams asking you to build in the product".

Reply:
Got it! This is actually a task for me – sorry if that wasn't clear. I need to get Gregg to approve the plan, then I need to work with Sales-Ops and Legal to make sure we're set up so AEs can actually use it. (Given that, I am going to leave it in)